docs/developing-locally-docker.rst

```diff
@@ -86,6 +86,20 @@ To migrate your app and to create a superuser, run::
 
 Here we specify the ``django`` container as the location to run our management commands.
 
+Setup your email backend
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+django-allauth sends an email to verify users (and superusers) after signup and login (if they are still not verified). To send email you need to `configure your email backend`_
```

Comment:
Hello Afrowave
All instructions in this document are only for local development using docker.